vehicle/skoda-enyaq.go

```diff
@@ -67,16 +67,16 @@ func NewEnyaqFromConfig(other map[string]interface{}) (api.Vehicle, error) {
 	)
 
 	if err == nil {
-		v.fromVehicle(vehicle.Name, float64(vehicle.Specification.Battery.CapacityInKWh))
+		uri := fmt.Sprintf("%s/v2/garage/vehicles/%s", skoda.BaseURI, vehicle.VIN)
+		err = api.GetJSON(uri, &vehicle)
```

Comment:
Warum brauchts den wenn die Fahrzeugliste vorher schon abgerufen ist? Falls es ihn braucht-> in api.Vehicle(vin) verpacken?

Reply:
Die Fahrzeugliste liefert jetzt nicht mehr direkt die Details zum Fahrzeug, sondern nur einen abgespeckten View pro Fahrzeug. Für die weiteren Details muß jetzt dieser Aufruf pro Fahrzeug erfolgen.

Hilf mir kurz - ich finde api.Vehicle(vin) nirgends. ❓

Reply:
Ich wollte nur vorschlagen, den Aufruf als 

    func (v *API) VehicleDetails(vin string) ...

zu implementieren, nicht an dieser Stelle.

Reply:
Sollte jetzt passen?

Reply:
Perfekt. Ein Check noch: connect/params.go ist jetzt nicht mehr genutzt da API ersetzt wird- der Ordner darf noch verschwinden?

Reply:
Ich glaube ja. Müsste jetzt über sein. Ich stell nachher einen _chore_ dafür ein.

Reply:
Ist schon erledigt 👍🏻